documentation/volume-tracking/docs/setting-up.md

```diff
@@ -0,0 +1,267 @@
+# Setting up for Development
+
+This section discusses setting up the volume tracking stack for local development. Setting up the end-to-end process involves several tools and technologies and therefore setting up involves spinning up [Docker](https://www.docker.com/) containers along with the application(s). Using containers helps the process to run the entire infrastructure stack in the development machine and makes setting up quite easy with ready-made scripts and manifests.
+
+## Overview
+
+Setting up the volume tracking stack involves running the following applications.
+
+1. `tol-lab-share`
+2. `traction-service`
+
+The third component used, as described in other sections, is MultiLIMS Warehouse. It could be set up locally, but since we could easily use the UAT instance of the warehouse, this document does not aim to discuss setting up a local warehouse instance. If required, it is possible to set up the warehouse by running a local instance of `unified_warehouse` which is documented in its [repository README](https://github.com/sanger/unified_warehouse).
+
+???+ info
+
+    At the time of this writing, there are only two running instances of MultiLIMS Warehouse. These instances are in following environments.
+
+    1. UAT/Training: The broker is accessible through the [management portal UI](http://ware-uat.psd.sanger.ac.uk:15672/#/).
```

Comment:
Probably don't want these addresses in our public docs

Reply:
Shall we change access to this set up page to private/internal? @dasunpubudumal ?

Reply:
If we are to do so, we'll need GitHub Enterprise Cloud (which is not free). I think for no we can remove the link and say something like "The broker is accessible through the management portal UI".